src/flexbox.rs

```diff
@@ -106,17 +106,17 @@ struct AlgoConstants {
     inner_container_size: Size<f32>,
 }
 
-impl Taffy {
+impl<T> Taffy<T> {
```

Comment:
Does this imply that we cannot blend measure function types within the same tree?

Reply:
In a way yes. Although the type of **stored data** has to much. In theory you can use several `Measure` implementation for the same tree.

I think that is probably acceptable in most cases, If you need to "blend" then it is possible to represent stored data as enum

```rs
enum MeasureData {
 Text { id: TextId; font_size: u8 },

 SomeOtherType(SomeMetadata) // <----
}
```